src/screens/HomeScreen.js

```diff
@@ -235,6 +240,7 @@ export const HomeScreen = ({ navigation, route }) => {
     {
       buttonTitle: buttonPointsOfInterestAndTours,
       fetchPolicy,
+      limit: limitPOI,
```

Comment:
should be `limitPointsOfInterestAndTours` to match the structure from news and events. i will adjust that.